modules/backend_checks/lambda.tf

```diff
@@ -0,0 +1,100 @@
+resource "aws_lambda_function" "backend_check_lambda" {
+  function_name = "tdr-${var.check_name}-${var.environment}"
+  runtime = "java8"
+  handler = "uk.gov.nationalarchives.tdr.${var.check_name}.RequestHandler::handleRequest"
+  role = aws_iam_role.backend_check_lambda_role.arn
+  timeout = 30
+  # seconds
```

Comment:
It looks a bit odd for this comment to be on the line below the thing it's commenting. It'd be clearer if it was either on the same line or above. I guess that eventually we might want to extract this to a top-level config file anyway?